src/components/editions/styles.ts

```diff
@@ -0,0 +1 @@
+export const editionsArticleWidth = '28rem';
```

Comment:
Don't all the places this is used also add `rem` to the end? So the result would be `28remrem`?

Reply:
🤦 file hadn't saved / committed. Should be there now,